checks.d/riakcs.py

```diff
@@ -107,8 +114,16 @@ def _get_stats(self, s3, aggregation_key):
 
         return stats
 
-
-    # We need this as the riak cs stats page returns json with duplicate keys
     @classmethod
     def load_json(cls, text):
-        return json.JSONDecoder(object_pairs_hook=multidict).decode(text)
+        data = json.loads(text)
+        if "legend" in data:
+            # riak cs before v2.1 had duplicate keys
+            data = json.JSONDecoder(object_pairs_hook=multidict).decode(text)
+        return data
+
+
+STATS_METHODS = {
+    "one": "count",
+    "total": "monotonic_count",
```

Comment:
As I'm reviewing these again and after reviewing the stats reported in my test, I think `total` maybe should use `gauge` rather than `monotonic_count`. Thoughts?